weave/flow/dataset.py

```diff
@@ -49,6 +52,19 @@ def from_calls(cls, calls: Iterable[Call]) -> Self:
         rows = [call.to_dict() for call in calls]
         return cls(rows=rows)
 
+    @classmethod
+    def from_pandas(cls, df: "pd.DataFrame") -> Self:
```

Comment:
isn't the standard term `from_df` and `to_df`? maybe i am wrong. Can you point to any existing apis that are similar?

Reply:
No, `to_pandas` is less ambiguous:
- https://docs.xarray.dev/en/latest/generated/xarray.DataArray.to_pandas.html
- https://docs.pola.rs/api/python/stable/reference/dataframe/api/polars.DataFrame.to_pandas.html
- https://docs.rapids.ai/api/cudf/latest/user_guide/api_docs/api/cudf.dataframe.to_pandas/
- https://networkx.org/documentation/networkx-1.10/reference/generated/networkx.convert_matrix.to_pandas_dataframe.html